pass-manager-svc/migration/src/main/kotlin/com/example/passmanagersvc/configuration/MongockConfiguration.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.passmanagersvc.configuration
```

Comment:
the file location `pass-manager-svc/migration/src/main/kotlin/com/example/passmanagersvc/configuration/`
but package `com.example.passmanagersvc.configuration`.

So, the package should be `com.example.passmanagersvc.migration.configuration`